hopsworks-persistence/src/main/resources/META-INF/persistence.xml

```diff
@@ -151,7 +152,9 @@
     <validation-mode>NONE</validation-mode>
     <properties>
       <property name="eclipselink.target-database" value="MYSQL"/>
-      <!--      <property name="eclipselink.logging.level" value="FINE"/>
+      <!--
+      <property name="eclipselink.profiler" value="PerformanceMonitor"/>
+      <property name="eclipselink.logging.level" value="FINE"/>
```

Comment:
what's the purpose of this change?

Reply:
To enable it when we want to monitor sql performance